lib/assets/uploader.js

```diff
@@ -0,0 +1,176 @@
+
+function setApplicationName(name, version) {
+  window.apiVideoFlutterUploader.params.application = {
+    name,
+    version
+  }
+}
+
+function setChunkSize(chunkSize) {
+  window.apiVideoFlutterUploader.params.chunkSize = chunkSize;
+}
+
+function getProgressiveSession(sessionId) {
+  return window.apiVideoFlutterUploader.progressiveSessions[sessionId];
+}
+
+function setProgressiveSession(sessionId, progressiveSession) {
+  window.apiVideoFlutterUploader.progressiveSessions = window.apiVideoFlutterUploader.progressiveSessions || {};
+  window.apiVideoFlutterUploader.progressiveSessions[sessionId] = progressiveSession;
+}
+
+function storeStandardUploader(uploader) {
```

Comment:
`setProgressiveSession` and `storeStandardUploader` have the same purpose but does not have a sibling name. Is there a reason for that?

Reply:
It's not quite the same, but I'll at least homogenize the names